PWGDQ/Tasks/tableReader_withAssoc.cxx

```diff
@@ -1254,6 +1263,9 @@ struct AnalysisSameEventPairing {
 
   HistogramManager* fHistMan;
 
+  o2::analysis::DQMlResponse<float> dqMlResponse;
+  std::vector<float> outputMlPsi2ee = {}; // TODO: check this is needed or not
```

Comment:
Please use the same naming conventions as the rest of the file (variable name starts with 'f', since it is a class data member)

Reply:
Sure, I understand. Although the `f` was used in AliPhysics because it was based on ROOT classes. That is not the case for O2.